CONFIGURATION.md

```diff
@@ -9,8 +9,13 @@ A sample configuration can be found at [config-sample.json](https://github.com/a
 | log_level  | Y        | String | Broker Log Level (DEBUG, INFO, ERROR, FATAL)
 | username   | Y        | String | Broker Auth Username
 | password   | Y        | String | Broker Auth Password
+| master_password_seed | Y        | String | Seed to generate DB instances master passwords
+| broker_name | Y        | String | RDS broker name used to tag instances for identification
```

Comment:
These values aren't in the top-level config block, they're under the `rds_config` section below.